PendingReleaseNotes

```diff
@@ -324,3 +324,10 @@
 
 * librbd now inherits the stripe unit and count from its parent image upon creation.
   This can be overridden by specifying different stripe settings during clone creation.
+
+* The balancer on by default in the upmap mode. Since the upmap mode requires
+  ``require_min_compat_client`` luminous, we are also changing the default value
+  of ``mon_osd_initial_require_min_compat_client`` from jewel to luminous. It is
```

Comment:
mon_osd_initial... isn't documented, better to explain its effect than to spell it out here